cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -539,11 +539,11 @@ core::PlanNodePtr SubstraitToVeloxPlanConverter::toVeloxPlan(const ::substrait::
     }
   }
 
-  std::vector<std::string> writePath;
-  writePath.reserve(1);
-  VELOX_CHECK(writeRel.named_table().names().size() == 1)
-  for (const auto& name : writeRel.named_table().names()) {
-    writePath.emplace_back(name);
+  std::string writePath;
+  if (writeFilesTempPath_.has_value()) {
+    writePath = writeFilesTempPath_.value();
+  } else {
+    writePath = "";
```

Comment:
Do we need to add a check that the `writePath `cannot be empty?

Reply:
added